src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -714,6 +781,152 @@ AS
     VALUES
         (@studyKey, @seriesKey, @instanceKey, @studyInstanceUid, @seriesInstanceUid, @sopInstanceUid, @newWatermark, @initialStatus, @currentDate, @currentDate)
 
+    -- Insert Custom Tags
+    -- String Key tags
+    IF EXISTS (SELECT 1 FROM @stringCustomTags)
```

Comment:
>IF EXISTS (SELECT 1 FROM @stringCustomTags) [](start = 4, length = 43)

It would be useful in the future if a few comments on what this merge is achieving are added. For example, why filter on tagStatus? How is tagLevel being used to decide to store series/instance keys? etc.

Reply:
will add